frontend/inbox/src/pages/Contacts/ContactDetails/index.tsx

```diff
@@ -56,7 +55,7 @@ const ContactDetails = (props: ContactDetailsProps) => {
   }, [cancelEditing]);
 
   const handleUpdateContact = () => {
-    updateContactDetails(conversationId, {id: contact.id, via: {email, phone}, title, organizationName: organization});
+    updateContactDetails({id: contact.id, via: {email, phone}, title, organizationName: organization});
   };
```

Comment:
I removed the `conversationId` from the payload to update contact details because we're sorting contacts by contact id.